README.md

```diff
@@ -44,5 +44,11 @@ currently specified in the `.gitmodules` file.
 For detailed walkthroughs of git submodule usage, see the
 [Git Tools documentation](https://git-scm.com/book/en/v2/Git-Tools-Submodules).
 
+## Building the Library
+Use the provided scripts `build-release.sh` and `build-debug.sh`. Ensure that `cmake` is installed first.
+
+### On macOS
+`cmake` can be installed using [Homebrew](https://brew.sh/): `brew install cmake`
```

Comment:
This is good information but makes me wonder if we need to provide corresponding instructions for Linux/Windows?

Reply:
I'm taking a stone soup approach here :) and general install instructions for linux are "install from your package manager whatever it is and how do you not have cmake on linux?" right?

Reply:
> general install instructions for linux are "install from your package manager whatever it is and how do you not have cmake on linux?" right?

Linux has newbies too! And not every linux distro comes with `build-essentials` or equivalent preinstalled. It never hurts to have friendly/complete documentation.